package/Makefile

```diff
@@ -81,7 +81,7 @@ endif
 ifndef SDK
   $(curdir)//compile = $(STAGING_DIR)/.prepared $(BIN_DIR)
 ifneq ($(CONFIG_USE_APK),)
-  $(curdir)/compile: $(curdir)/system/apk/host/compile $(BUILD_KEY_APK_SEC) $(BUILD_KEY_APK_PUB)
```

Comment:
BTW I've hit that as well and come up with the following solution (untested fully yet) https://github.com/ynezz/openwrt/commit/ff6d450321af205de6820b5b4ac2b392b9cdecc4

Reply:
Lets see how it goes https://buildbot.staging.openwrt.org/images/#/builders/3/builds/2

Reply:
@ynezz did you verified that this works by removing the hostpkg apk build_dir and apk tool from staging_dir?

$(curdir)//compile is actually different to $(curdir)/compile ?

Reply:
@Ansuel It's a hack to have two different Make targets for the same file...

Reply:
@ynezz Looks good now, doesn't it? In production we will want to provision the key instead of generating a new one for every single build. Or find a way to skip signing at the pkgbuild stage and have another instance not running any 3rd-party code do that part separately -- at the cost of having to upload all packages to that other instance...

Reply:
if we really want to to sign each package... maybe there the tool adbsign with --reset-signature that we maybe use to sign .apk later when they are packaged... but i still have to try that...

Currently busy with damn apk not being able to handle long filename in the script.tar that is a big red flag...

Example:
![image](https://github.com/openwrt/openwrt/assets/20289090/504b44a5-6e3f-4f2f-b8f7-67c9f1c2cb32)

Reply:
> Looks good now, doesn't it?

Yep, at least it builds now.

> Or find a way to skip signing at the pkgbuild stage and have another instance not running any 3rd-party code do that part separately

Keys on disk = not your keys anymore :-) Thus I've started [Using Nitrokey 3A Mini for build artifact signing key storage](https://lists.openwrt.org/pipermail/openwrt-devel/2023-May/041044.html).